packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -8,30 +8,54 @@ const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
   constructor(options = {}) {
-    this.formFactor = options.formFactor;
+    const { formFactor, theme } = options;
+
+    this.formFactor = formFactor;
+    this.theme = theme || 'terra-default-theme';
```

Comment:
In [#464](https://github.com/cerner/terra-toolkit/pull/464/files#diff-a6ba3c1d16c08e58e7ee3994f3ab541b63b2a3a8dafb0de9daed2c8090c948d9) I'm exposing the entire service options object thru `Terra.serviceOptions`. I should probably default `Terra.serviceOptions.theme` to `terrra-default-theme` to keep them in sync. Would it be better to default the theme to `terrra-default-theme` in wdio.config.js or are we concerned that this config may get overridden by consumers.

Reply:
I think the TerraService should default the option if it needs a default rather than requiring a theme be set in the wdio.config. 

In this case the default is more of a best practice/placeholder to keep concepts in sync. The `terra-default-theme` currently has no functionality in this service. It's solely to compare against whether or not the current theme is the lowlight theme. Setting `terra-default-theme` at the wdio.config level wouldn't have any change in the behavior of the service